### DISCUSSION TITLE
Patch Release 2.8.7

### DISCUSSION THREAD

**Discussion by APickledWalrus** [SkriptLang/Skript#6787] [General]
# Skript 2.8.7

We're releasing 2.8.7 to fix some important issues that made their way into 2.8.6. We expect this to be the final version for Skript 2.8. You can report any issues through our [issue tracker](https://github.com/SkriptLang/Skript/issues).

Happy Skripting!

### Bug Fixes
- [#6757](<https://github.com/SkriptLang/Skript/pull/6757>) Fixed an error that could occur when attempting to obtain the potion effects of a plain potion.
- [#6758](<https://github.com/SkriptLang/Skript/pull/6758>) Fixed arithmetic-related errors on Java 8 and when performing Vector-Vector multiplication.
- [#6760](<https://github.com/SkriptLang/Skript/pull/6760>) Fixed several particle definition conflicts that made it harder to use certain items and entity types with variables.
- [#6763](<https://github.com/SkriptLang/Skript/pull/6763>) Fixed an issue where reloading scripts with commands could cause an exception on Paper 1.20.5+.
- [#6764](<https://github.com/SkriptLang/Skript/pull/6764>) Fixed an issue where fireworks could not be spawned using the spawn effect/section.
- [#6777](<https://github.com/SkriptLang/Skript/pull/6777>) Fixed an issue with the text representation of the case expression.

[Click here to view the full list of commits made since 2.8.6](<https://github.com/SkriptLang/Skript/compare/2.8.6...2.8.7>)

### Java 11
From Skript 2.9.0 onwards, we will be requiring a minimum Java version of 11 to run.

### Help Us Test
We have an [official Discord community](https://discord.gg/ZPsZAg6ygu) for beta testing Skript's new features and releases.

### Thank You

Special thanks to the contributors whose work was included in this version:
- @APickledWalrus 
- @ShaneBeee

As always, if you encounter any issues or have some minor suggestions, please report them at https://github.com/SkriptLang/Skript/issues.
If you have any bigger ideas or input for the future of Skript, you can share those too at https://github.com/SkriptLang/Skript/discussions.

<hr /><em>This discussion was created from the release <a href='https://github.com/SkriptLang/Skript/releases/tag/2.8.7'>Patch Release 2.8.7</a>.</em>

**Comment by knasigma** [comment]
Pls update 1.21
